plugins/hbase_logs.yaml

```diff
@@ -0,0 +1,132 @@
+      - end
+    default: end
+template: |
+  receivers:
+    {{ if .enable_master_log }}
+    filelog/master:
+      include:
+        {{ range $fp := .master_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+          line_start_pattern: '^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}\s+|^\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}'
+      start_at: {{ .start_at }}
+      operators:
+        - type: regex_parser
+          if:  '$record matches "^\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2},\\d{3}\\s+"'
+          regex: '^(?P<timestamp>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}),\d{3}\s+(?P<hbase_severity>[A-Z]*)\s*?\[(?P<thread>[^\]]*)\]\s(?P<hbase_source>[^:]*):\s(?P<message>(?s).*)\n'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%Y-%m-%d %H:%M:%S'
+          severity:
+            parse_from: attributes.hbase_severity
+            mapping:
+              warning: warn
+              emergency: fatal
+          if: '$record matches "^\\w+ \\w+ \\d{2} \\d{2}:\\d{2}:\\d{2} \\w+ \\d{4}"'
+          regex: '^(?P<timestamp>\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}) (?P<message>(?s).*)\n'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%a %h %d %H:%M:%S %Z %Y'
+    {{ end }}
+
+    {{ if .enable_region_log }}
+    filelog/region:
+      include:
+        {{ range $fp := .region_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+       line_start_pattern: '^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}\s+|^\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}'
+      start_at: {{ .start_at }}
+      operators:
+        - type: regex_parser
+          if:  '$record matches "^\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2},\\d{3}\\s+"'
+          regex: '^(?P<timestamp>\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2}),\d{3}\s+(?P<hbase_severity>[A-Z]*)\s*?\[(?P<thread>[^\]]*)\]\s(?P<hbase_source>[^:]*):\s(?P<message>(?s).*)\n'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%Y-%m-%d %H:%M:%S'
+          severity:
+            parse_from: attributes.hbase_severity
+            mapping:
+              warning: warn
+              emergency: fatal
+          if: '$record matches "^\\w+ \\w+ \\d{2} \\d{2}:\\d{2}:\\d{2} \\w+ \\d{4}"'
+          regex: '^(?P<timestamp>\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}) (?P<message>(?s).*)\n'
+          timestamp:
+            parse_from: attributes.timestamp
+            layout: '%a %h %d %H:%M:%S %Z %Y'
+    {{ end }}
+
+    {{ if .enable_zookeeper_log }}
+    filelog/zookeeper:
+      include:
+        {{ range $fp := .zookeeper_log_path }}
+        - '{{ $fp }}'
+        {{ end }}
+      multiline:
+       line_start_pattern: '^\d{4}-\d{2}-\d{2} \d{2}:\d{2}:\d{2},\d{3}\s+|^\w+ \w+ \d{2} \d{2}:\d{2}:\d{2} \w+ \d{4}'
+      start_at: {{ .start_at }}
+      operators:
+        - type: regex_parser
+          if:  '$record matches "^\\d{4}-\\d{2}-\\d{2} \\d{2}:\\d{2}:\\d{2},\\d{3}\\s+"'
```

Comment:
beginning of if expression should remove the `^` character

Reply:
Just made that change